src/topology/sheaves/skyscraper.lean

```diff
@@ -158,4 +169,18 @@ def skyscraper_presheaf_stalk_of_not_specializes_is_terminal
   [has_colimits C] {y : X} (h : ¬p₀ ⤳ y) : is_terminal ((skyscraper_presheaf p₀ A).stalk y) :=
 is_terminal.of_iso terminal_is_terminal $ (skyscraper_presheaf_stalk_of_not_specializes _ _ h).symm
 
+variables [Π (U : opens (Top.of (punit : Type u))), decidable (punit.star ∈ U)]
```

Comment:
```suggestion
```
You don't need this instance to state the lemma and define `skyscraper_sheaf` below, so if you need the instance you'd just use `by classical; exact ...`.